server/handlers.go

```diff
@@ -458,7 +470,7 @@ func (s *LivepeerServer) setOrchestratorConfigHandler(client eth.LivepeerEthClie
 	}))
 }
 
-func (s *LivepeerServer) setOrchestratorPriceInfo(broadcasterEthAddr, pricePerUnitStr, pixelsPerUnitStr string) error {
+func (s *LivepeerServer) setOrchestratorPriceInfo(broadcasterEthAddr, pricePerUnitStr, pixelsPerUnitStr, currency string) error {
```

Comment:
I tried and for me it didn't work. Here's what I tried (you can try it with the local dev setup, with some mocking :) ).

1. I started livepeer with `-pricePerUnit 0.0000000000007USD`
2. Tried with `livepeer_cli`
```
 *-----------------------------*--------------------------------*
|        Base price per pixel |           70000000000000 wei / |
|                             |            360458402809 pixels |
*-----------------------------*--------------------------------*
| Base price for broadcasters |                                |
*-----------------------------*--------------------------------*
```
All good.

3. Then, I executed from `livepeer_cli`, `20 Set price for broadcaster`, and I've set the price per broadcaster and also the default one
4. Tried with `livepeer_cli`
*-----------------------------*---------------------------------------------*
|        Base price per pixel |                            0 wei / 1 pixels |
*-----------------------------*---------------------------------------------*
| Base price for broadcasters | 0xc3c7c4c8f7061b7d6a72766eee5359fe4f36e61e: |
|                             |                                0 per pixel  |
*-----------------------------*---------------------------------------------*

This looks wrong, not sure it's related to your change, but it can be. Could you please play with it more?

Reply:
Will do. I believe I'll also need to update `livepeer_cli` to support getting input on currency that it'd like to update to

Reply:
Ok I've implemented (and tested e2e) the `livepeer_cli` support here: https://github.com/livepeer/go-livepeer/pull/2981/commits/13bc0a6b50ed85fe56e48284b67ef31465f5c27d

I've tested setting configs for B, O (base and per B). Also tested with some invalid values as well as switching back and forth between wei/usd and checked we don't keep the price feed loop running. 

I was not able to test the catalyst build with a pending go-livepeer version though. I get errors like this [1] on the steps that download go-livepeer. Do you know if I have to trigger something in this repo for those on catalyst to work?

[1]
```
0327 00:19:58.855627 2263829 utils.go:91] Downloading https://github.com/livepeer/playbacklog_uploader/releases/download/v1.3.2/livepeer-mist-bigquery-uploader-linux-arm64.tar.gz.sig
I0327 00:19:58.893342 2263829 utils.go:100] Response statusCode=404
F0327 00:19:58.893403 2263829 downloader.go:244] failed to download livepeer: HTTP 404 while downloading https://build.livepeer.live/go-livepeer/13bc0a6b50ed85fe56e482[84](https://github.com/livepeer/catalyst/actions/runs/8444445609/job/23129999424#step:5:85)b67ef31465f5c27d/livepeer-linux-arm64.tar.gz
```